tests/helpers.py

```diff
@@ -118,11 +118,11 @@ def setup_class(self):
     def teardown_class(self):
         shutil.rmtree(self.temp_dir)
 
-    def check_stat(self, check, idata, name):
+    def check_stat(self, check, idata):
         group = idata.posterior
         for var, stat, value, bound in check:
             s = stat(group[var].sel(chain=0), axis=0)
-            close_to(s, value, bound, name)
+            assert_array_less(np.abs(s.values - value), bound)
```

Comment:
Can we use `assert_allclose` here?

Reply:
No, because ``bound`` is an array.

Reply:
`assert_allclose` works with arrays as well

Reply:
Ahh the tolerance is the array? Is that ever used, or are we passing scalars all the time?

Reply:
It's used. The test is used for multivariate arrays, and the tolerance is set to match the expected error from MC samples.

Reply:
My bad again then! Sorry for the trouble

Reply:
No trouble at all, thanks for checking!